idlpy/src/types.c

```diff
@@ -185,42 +229,68 @@ emit_field(
         }
     }
 
-    idlpy_ctx_printf(ctx, "\n    %s: %s", name, type);
+    //Reserved Python keywords support (Issue 105):
+    const char *nameField = name;
+    const char *nameFieldModified=get_c_name(name);
+    if (nameFieldModified)
+        nameField =  nameFieldModified;
+    
+    idlpy_ctx_printf(ctx, "\n    %s: %s", nameField, type);
+
+    if (nameFieldModified) {
+        idlpy_ctx_printf(ctx, "\n    annotate.member_name(\"%s\",\"%s\")", nameFieldModified,name);
+
+    }
 
     if (idl_is_member(parent)) {
+
+        //printf("emit_field idl_is_member2 name : %s , type %s \n", name , type);
```

Comment:
Please don't leave commented code in the PR.

Reply:
After reviewing it today, all the printfs we have added have been deleted.